index.js

```diff
@@ -1182,20 +1189,37 @@ module.exports = function (log, indexesPath) {
     return a.timestamp > b.timestamp
   }
 
-  function sortedByTimestamp(bitset, descending) {
+  function compareSeqAscending(a, b) {
+    return b.seq > a.seq
+  }
+
+  function compareSeqDescending(a, b) {
+    return a.seq > b.seq
+  }
+
+  function sortedBy(bitset, descending, sortBy) {
     updateCacheWithLog()
     const order = descending ? 'descending' : 'ascending'
+    const sortedCache = sortBy === 'arrival' ? sortedSeqCache : sortedTSCache
+    const comparer =
+      sortBy === 'arrival'
+        ? descending
+          ? compareSeqDescending
+          : compareSeqAscending
+        : descending
+        ? compareDescending
+        : compareAscending
```

Comment:
Could `compareDescending`/`Ascending` be renamed to `compareTSDescending`/`Ascending` so that it's consistent with the naming of `compareSeqDescending`?